users/serializers.py

```diff
@@ -207,5 +214,81 @@ def get_redirect_url(self):
         """
         return f"{settings.SLACK_AUTH_LOGIN_REDIRECT}{self.token.token}/"
 
+class AddPasswordSerializer(serializers.Serializer):
```

Comment:
@melchiaballe You can merge this class with the other one `ChangePasswordSerializer`. they are almost similar to each other so it's not worth it to create 2 versions of them.

You can merge them as `PasswordSerializer`. add a checker/condition in the `validate()` to check if the `request.user` has a password. If None, then do not check for old password, If the user has a password already, check for old password.